incubator/fluentd-cloudwatch/templates/psp.yaml

```diff
@@ -1,5 +1,5 @@
 {{- if .Values.rbac.pspEnabled }}
-apiVersion: extensions/v1beta1
```

Comment:
maybe use capabilities instead to make it dynamic so you can target older and newer apis with same chart: https://helm.sh/docs/chart_template_guide/builtin_objects/ ?